TLM/TLM/Manager/Impl/RoutingManager.cs

```diff
@@ -733,319 +732,356 @@ public class RoutingManager
+                                                    "and is connected with previous lane. adding as lane connection lane.",
+                                                    prevSegmentId,
+                                                    prevLaneIndex,
+                                                    prevLaneId,
+                                                    isNodeStartNodeOfPrevSegment,
+                                                    nextLaneId,
+                                                    nextLaneIndex);
+                                            }
+                                        } else {
+                                            if (extendedLogRouting) {
+                                                Log._DebugFormat(
+                                                    "RoutingManager.RecalculateLaneEndRoutingData({0}, {1}, " +
+                                                    "{2}, {3}): nextLaneId={4}, idx={5} has outgoing connections " +
+                                                    "but is NOT connected with previous lane",
+                                                    prevSegmentId,
+                                                    prevLaneIndex,
+                                                    prevLaneId,
+                                                    isNodeStartNodeOfPrevSegment,
+                                                    nextLaneId,
+                                                    nextLaneIndex);
+                                            }
+                                        }
+                                    } else {
+                                        bool goodTurnAngle = TrackUtils.CheckSegmentsTurnAngle(
+                                            sourceSegment: ref nextSegment,
+                                            targetSegment: ref prevSegment,
+                                            nodeId: nodeId);
+                                        bool nextIsTrackOnly = nextLaneInfo.IsTrackOnly();
+                                        bool similarLaneCountMatches = prevSimilarLaneCount == nextSimilarLaneCount;
+                                        bool stayInlane = nextIsTrackOnly & similarLaneCountMatches & goodTurnAngle;
+
+                                        bool connected;
+                                        if (stayInlane) {
+                                            if (extendedLogRouting) {
+                                                Log._Debug($"similar track networks: Preventing lane changes.");
+                                            }
+                                            connected = outerSimilarLaneIndexMatches;
+                                        } else {
+                                            connected = goodTurnAngle;
+                                        }
```

Comment:
```csharp
connected = stayInLane
    ? outerSimilarLaneIndexMatches
    : goodTurnAngle;

Log._DebugIf(extendedLogRouting && stayInLane, ...); // auto-removed from non-DEBUG builds
```

Reply:
`extendedLogRouting` is `const` in release builds. so `if(extendedLogRouting)` will be ignored in release builds.